### ISSUE TITLE
Account Manager Tool

### ISSUE THREAD

**Issue by Eastrall** [Eastrall/Hellion#33]
Let's think about a tool that can manage all the accounts of you server. The tool we provide the possibility to : 
- View all account informations
- Create a new account
- Edit an account
- Delete an account

Basicaly it's a simple CRUD tool for accounts. It will provide an alternative for the server administrator to manage accounts instead of searching in MySQL Workbench or other tools.

**Comment by raventh1984** [comment]
Well an tools to manage accounts is not an priority at this time.

Cause you can simple make an .php file for it. And use it only on localhost.


I am looking arround in the source to understand how it all works so i can get my hands down on it :D.


I am a beginner at c#. But an fast learner.

So best thing to tackle first is the battle system.


________________________________
Van: Filipe GP <notifications@github.com>
Verzonden: dinsdag 14 februari 2017 09:53
Aan: Eastrall/Hellion
CC: Subscribed
Onderwerp: [Eastrall/Hellion] Account Manager Tool (#33)


Let's think about a tool that can manage all the accounts of you server. The tool we provide the possibility to :

  *   View all account informations
  *   Create a new account
  *   Edit an account
  *   Delete an account

Basicaly it's a simple CRUD tool for accounts. It will provide an alternative for the server administrator to manage accounts instead of searching in MySQL Workbench or other tools.

**Comment by Eastrall** [comment]
I agree that the tools are not a prioprity for now. Just saying that this could be a good tool to develop in order to help server administrator in the futur.

Right now I agree that the Battle system is more important, because it's the main system of FlyFF.

**Comment by davedevils** [comment]
Why not just make all in tools ?

Like All in Tools - resource and All in Tools - server

All in Tools - resource : 
-> edit resource 
-> edit npc 
-> edit quest 
ect...


All in Tools - server : 
-> Create account
-> ban player
-> show log
-> get stat about server
-> send item to a player
etc...

**Comment by Eastrall** [comment]
For the server it could be a good thing to do. A all-in-one tool to manage the server. For the resources I don't really now if a all-in-one tool would be nice. Because each tool are targeting a specific resource. For example the ShopEditor will have a different design than the Account Editor, or the QuestEditor. It's all about ergonony and efficiency.

**Comment by davedevils** [comment]
I will show you what idea i have about this 

The Tools for edit : 
http://img4.hostingpics.net/pics/715523Capturedu20170320001249.png

The tools for use server : 
http://img4.hostingpics.net/pics/444971Capturedu20170320001659.png


It's will be more fast to work for dev or admin and can edit more fast all

**Comment by Eastrall** [comment]
Indeed, having a tool to manage the server would be great to manage a server for the administrators. But I'm not really convinced about a all-in-one tool for the resource editor.

The real hard work about a tool such as this, is to provide an intuitive interface and ergonomic, because the tool must be scalable and maintainable, to add/update features easily.

We need to think about this tool later 😄

**Comment by davedevils** [comment]
if i post now it's for you can thinks about that because make lot's of tools is so old , and you want use modern and "new" consept , so go make it all modern not just 2/3 part ^^
